homeassistant/components/climate/ecobee.py

```diff
@@ -382,13 +383,3 @@ def hold_preference(self):
             return default
         else:
             return 'nextTransition'
-
-    # Sleep mode isn't used in UI yet:
```

Comment:
@nordlead2005 Actually, no climate can be a hold. A climate can be referenced in a temperature hold, which means that certain values for the desired state are taken from that climate. For example, the fan operation or the participating sensors might be taken from the climate, rather than separately defined.